loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -423,67 +465,105 @@ static void binary_update_same_version_test(void)
 static void binary_update_new_version_test(void)
 {
 	int ret;
-	char filepath[CONFIG_PATH_MAX];
+	char filepath[BINARY_PATH_LEN];
 	binary_update_info_t pre_bin_info;
 	binary_update_info_t cur_bin_info;
 
-	binary_update_getinfo(APP_NAME, &pre_bin_info);
+	ret = binary_update_getinfo(APP_NAME, &pre_bin_info);
+	if (ret != OK) {
+		return;
```

Comment:
How about printing the log about fail case?

Reply:
binary_update_getinfo prints error message.